### PR TITLE
Octopus Cloud and Infrastructure overview page updates

### PR DESCRIPTION
**Pull request by tonykelly-octopus**
Copy and image updates to the Octopus Cloud and Infrastructure overview pages.

### Octopus Cloud
image

### Infrastructure
image

### Octopus Cloud
image
- The image isn't showing in the preview, but it is there. Takes a while to kick through.

image
- New page: content taken from the previous Octopus Cloud overview page

### Infrastructure
image
- The image isn't showing in the preview, but it is there. Takes a while to kick through.

### CODE REVIEW

**Comment by tonykelly-octopus** [comment]
Pushed up a change for a couple of the images

File: src/pages/docs/infrastructure/index.mdx [hunk 1]
```diff
@@ -13,40 +13,37 @@ hideInThisSection: true
...
+  - [Tentacle](/docs/infrastructure/deployment-targets/tentacle): A lightweight deployment agent that runs on [Windows](/docs/infrastructure/deployment-targets/tentacle/windows) or [Linux](/docs/infrastructure/deployment-targets/tentacle/linux) servers.
+  - [Target tag](/docs/infrastructure/deployment-targets#target-roles): A tag that defines a purpose for the deployment target, which can be used to deploy particular components to a set of targets.
 
 ## Learn more
 
 - [Environments](/docs/infrastructure/environments)
 - [Deployment targets](/docs/infrastructure/deployment-targets)
 - [Workers](/docs/infrastructure/workers)
 - [Accounts](/docs/infrastructure/accounts)
+- [Signing keys](/infrastructure/signing-key)
```
Comment (steve-fenton-octopus) [inline src/pages/docs/infrastructure/index.mdx:49]: This is the link that results in an error:

Received: "Failed to fetch http://[::1]:3000/infrastructure/signing-key due to Error: Expected a 200 OK response for page http://[::1]:3000/infrastructure/signing-key found on http://[::1]:3000/docs/infrastructure/·

Comment (steve-fenton-octopus) [reply src/pages/docs/infrastructure/index.mdx:49]:
```suggestion
- [Signing keys](/docs/infrastructure/signing-keys)
```

File: src/pages/docs/octopus-cloud/index.mdx [hunk 2]
```diff
@@ -14,104 +14,55 @@ import OctopusCloudInstance from 'src/shared-content/octopus-cloud/octopus-cloud
...
-
-## Reset the Octopus account password \{#reset-the-octopus-account-password}
-
-If you forget your username or password for your Octopus account or Octopus instance, you can reset them via Octopus.com
-
-1. Visit [Octopus.com/signin](https://Octopus.com/signin).
-1. Click **Forgot your password?**
-1. Reset your credentials and log into your Octopus account. You can now launch your Octopus instance from the accounts dashboard.
-
-## Invite users to your Octopus Cloud instance \{#inviting-octopus-cloud-users}
```
Comment (steve-fenton-octopus) [inline src/pages/docs/octopus-cloud/index.mdx:78]: This bookmark is referenced in short.io / the Octopus product, so the bookmark test is failing.

File: src/pages/docs/octopus-cloud/getting-started-with-cloud.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+## Reset the Octopus account password \{#reset-the-octopus-account-password}
+
+If you forget your username or password for your Octopus account or instance, you can reset them via octopus.com.
+
+1. Visit [octopus.com/signin](https://octopus.com/signin).
+2. Click **Forgot your password?**
+3. Reset your credentials and log into your Octopus account. You can now launch your Octopus instance from the accounts dashboard.
+
+## Invite users to your Octopus Cloud instance
```
Comment (steve-fenton-octopus) [inline src/pages/docs/octopus-cloud/getting-started-with-cloud.mdx:61]:
```suggestion
## Invite users to your Octopus Cloud instance \{#inviting-octopus-cloud-users}
```

**Review by steve-fenton-octopus — APPROVED** [review]
LGTM

**Pull request merged**